### PR TITLE
feat(component): add prop to toggle full input selection on click/focus

### PR DESCRIPTION
**Pull request by Blastius**

### CODE REVIEW

File: gh-pages-src/pages/Docs.vue [hunk 1]
```diff
@@ -188,6 +188,11 @@ export default {
         name: 'after-items-fixed-absolute',
         scopeProperties: {},
         description: 'Define a custom error appearance (fixed, absolute)'
+      },
+      {
+        name: 'select-text-on-focus',
```
Comment (iliyaZelenko) [inline gh-pages-src/pages/Docs.vue]: These are slots, not props. Props go to the documentation automatically. 😄

**Comment by iliyaZelenko** [comment]
I did not look at it much, but I already spent 5 hours today on this project, I got a little tired. 😃  But thanks for the PR, supporting the project is a very good idea!

**Comment by Blastius** [comment]
Code should be OK and up-to-date now

File: gh-pages-src/pages/Docs.vue (hunk 1 above)
Comment (Blastius) [reply gh-pages-src/pages/Docs.vue]: Cleaned up Docs.vue 😃

**Comment by iliyaZelenko** [comment]
Ok, I’ll look at it today or tomorrow, as I will be free, thank you!

**Comment by Blastius** [comment]
Hi @iliyaZelenko , have you had any time to check this PR?

**Comment by iliyaZelenko** [comment]
Yes, I forgot about it, thanks for the reminder. Today I’ll definitely look.

**Comment by iliyaZelenko** [comment]
Merged. Changed the code a bit, it seems to work. I wake up and promise that I will release a new version in the next 24 hours. This is a good idea! And thanks for most of the code in the implementation!

**Pull request merged**